tests/components/bluetooth_le_tracker/test_device_tracker.py

```diff
@@ -0,0 +1,62 @@
+"""Test Bluetooth LE device tracker."""
+
+from datetime import timedelta
+from unittest.mock import Mock, patch
+
+from homeassistant.components.bluetooth_le_tracker import device_tracker
+from homeassistant.components.device_tracker.const import (
+    CONF_SCAN_INTERVAL,
+    CONF_TRACK_NEW,
+    SOURCE_TYPE_BLUETOOTH_LE,
+)
+from homeassistant.const import ATTR_NOW, EVENT_TIME_CHANGED
+from homeassistant.util import dt as dt_util
+
+from tests.common import get_test_home_assistant
+
+
+def test_preserve_new_tracked_device_name():
```

Comment:
Please use a coroutine test and use `hass` fixture to get a home assistant instance.

Reply:
Done